client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -40,17 +42,23 @@ class ActivityLogDay extends Component {
 	 * @returns { object } Button to display.
 	 */
 	getRewindButton( type = '' ) {
-		return (
-			<Button
-				compact
-				disabled={ ! this.props.isRewindEnabled }
-				onClick={ this.handleClickRestore }
-				primary={ 'primary' === type }
-			>
-				<Gridicon icon="history" size={ 18 } />
-				{ ' ' }
-				{ this.props.translate( 'Rewind to this day' ) }
-			</Button>
+		const { allowRestore } = this.props;
+
+		return ( allowRestore
+			? (
```

Comment:
Probably would have gone with a
```
if ( ! allowRestore ) {
   return null;
}
...
```
For a better diff and less indent, but fine as it is.